test/adversary/adversary-code/database.mjs

```diff
@@ -85,10 +82,7 @@ const duplicateNullifier = async number => {
   try {
     const connection = await mongo.connection(MONGO_URL);
     const db = connection.db(OPTIMIST_DB);
-    const res = await db
-      .collection(TRANSACTIONS_COLLECTION)
-      .find({ transactionType: { $in: ['1', '2'] } })
-      .toArray();
+    const res = await db.collection(TRANSACTIONS_COLLECTION).find().toArray();
```

Comment:
we will need to filter this for transactions that only have nullifiers for this test